vmware/container-service-extension
# [VCDA-1268] CSE support for encrypted configuration file

- CSE supports encrypted configuration file

- As part of security requirements, this commit includes changes for the following
   * Encryption of configuration file
   * Decryption of configuration file
   * cse install, run, template, check, convert-cluster handling encrypted configuration
   * Option for skipping encryption
   * Exception handling for failed encryption/decryption

- Manual testing completed successfully
   cse encrypt, cse decrypt, cse run, cse install, cse template, cse convert-cluster, cse check

@andrew-ni @rocknes


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/container-service-extension/474)
